### ISSUE TITLE
Feature Request: QuickBooks Online Integration

### ISSUE THREAD

**Issue by trumblejoe** [almarklein/timetagger#316]
Any chance of adding a time-import to QuickBooks Online? Downloading and Uploading CSV files is it's own beast in terms of compliance but a direct-data import to a major bookkeeping product would be mint.

**Comment by almarklein** [comment]
Sure! Would you be able to provide a few lines of (anonymized) CSV that QuickBooks generates? I could then try to adjust the timetagger import dialog to recognize it, and let you test-drive it.

**Comment by trumblejoe** [comment]
Yes! I will send you something this week as time permits at work.

**Comment by trumblejoe** [comment]
Here is the REST API from QBO if it helps in the meantime.

https://developer.intuit.com/app/developer/qbo/docs/get-started

**Comment by trumblejoe** [comment]
Here we go, this has an example of the timesheet. Fairly straight-forward.
https://quickbooks.intuit.com/learn-support/en-us/employees-and-payroll/import-timesheet-as-a-csv-file/00/1109314

**Comment by almarklein** [comment]
Pasting here for simplicity:
<p>Prepare your Time Tracking data with the following template</p>

Name | Transaction Date | Customer | Service Item | Payroll item | Duration | Class | Billable | Notes
-- | -- | -- | -- | -- | -- | -- | -- | --
John Marks | 01/09/2020 | Nguyen, Tuan | Labor | Regular Pay | 1:25 |   |   |  
John Smith | 02/09/2020 | Nguyen, Tuan | Labor | Regular Pay | 2:00 |   |   |  
Karen Marks | 02/09/2020 | Links, John | Labor | Regular Pay | 1:25 |   |   |  
Kathy Linklater | 02/09/2020 | Links, John | Labor | Regular Pay | 1:25 |   |   |  
Johnny Deep | 02/09/2020 | Links, John | Labor | Regular Pay | 2:00 |   |   |  

<p> </p>

This is for importing data into QB, or is this also the format that it exports data to?

**Comment by trumblejoe** [comment]
Import format. I'm not sure of the export format.
